<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfLogNotificationsPrefWidget</class>
 <widget class="QWidget" name="WkfLogNotificationsPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通知</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>弹窗通知</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="notificationsBox">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="notificationLevel1">
        <property name="text">
         <string>致命</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="notificationLevel2">
        <property name="text">
         <string>致命错误</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="notificationLevel3">
        <property name="text">
         <string>致命错误警告</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>这些选项仅会在应用程序重新启动时才会停止显示弹出窗口.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignoreErrors">
        <property name="text">
         <string>忽略错误</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignoreWarnings">
        <property name="text">
         <string>忽略警告</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
